Description: increaste timeout for this test
 https://buildd.debian.org/status/fetch.php?pkg=inkscape&arch=mipsel&ver=1.2.2-1&stamp=1671553799&raw=0
 https://buildd.debian.org/status/fetch.php?pkg=inkscape&arch=mips64el&ver=1.2.2-1&stamp=1671566710&raw=0
 In the past it took ~60 seconds, but suddenly it timeouted after 180.
 Let's try increasing this.
Author: Mattia Rizzolo <mattia@debian.org>
Forwarded: not-needed
Last-Update: 2022-12-21

--- a/testfiles/CMakeLists.txt
+++ b/testfiles/CMakeLists.txt
@@ -105,6 +105,8 @@
     add_dependencies(tests ${testname})
 endforeach()
 
+# on mipsel this timeouted after 180s
+set_tests_properties(test_lpe PROPERTIES TIMEOUT 600)
 
 ### CLI rendering tests and LPE
 add_subdirectory(cli_tests)
